src/ui/public/agg_types/buckets/histogram.js

```diff
@@ -59,7 +59,15 @@ export const histogramBucketAgg = new BucketAggType({
       name: 'field',
       filterFieldTypes: 'number'
     },
-
+    {
+      /*
+       * This parameter can be set if you want the auto scaled interval to always
+       * be a multiple of a specific base.
+       */
+      name: 'intervalBase',
+      default: null,
+      write: () => {},
```

Comment:
This prevents writing `intervalBase` to DSL. Does this merit a comment or a test?

Reply:
Added an explicit test for that